app/views/foods/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1  align="center">Add new Foods</h1>
+<div class="col-lg-12" id="signup">
+  <%= form_with model: @food, url: user_foods_path(@user) do |form| %>
+   
+  
+    <div class="mb-3">
+    <%= form.text_field :name, placeholder: "Enter Your Name", class: "form-control" %>
+    </div>
```

Comment:
- [ ] It looks like there was a typo here and instead of Enter recipe foods name I see "Enter Your Name" 😁
<img width="651" alt="Screenshot 2022-07-29 at 21 28 14" src="https://user-images.githubusercontent.com/77038610/181812556-1838fd6d-a7b7-4459-9e42-4ed8f5a7f662.png">